fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -612,7 +612,12 @@ private void addLinkAndOptionsHttpHeaders() {
             servletResponse.addHeader("Accept-Patch", contentTypeSPARQLUpdate);
 
         } else if (resource() instanceof Container) {
-            options = "MOVE,COPY,DELETE,POST,HEAD,GET,PUT,PATCH,OPTIONS";
+            // MOVE/COPY/DELETE operations are not available on a RepositoryRoot resource
+            if (resource().getContainer() == null) {
+                options = "POST,HEAD,GET,PUT,PATCH,OPTIONS";
```

Comment:
Oh, I didn't realize that it doesn't work. I wonder why? Oh, well, it's certainly better not to advertise it if it doesn't work.